### DISCUSSION TITLE
No content in pages?

### DISCUSSION THREAD

**Discussion by sinnbeck** [filamentphp/filament#9001] [Help]
I am just trying out the panels builder in filament, but for some reason all pages are blank?
![image](https://github.com/filamentphp/filament/assets/13980140/fdeef03c-2228-4a75-b2d0-f6c69b1fc55a)

There is no vendor folder in my resources/views folder (I even tried deleting the views folder).

Had anyone experienced this before? I am unsure where to start debugging

**Comment by leandrocfe** [comment]
Is it a fresh install? Could you run?
```bash
php artisan filament:upgrade
```

**Reply by sinnbeck** [reply]
Not quite. I have used some filament components for the frontpage earlier.

I have tried installing filament again, as well as deleting the AdminPanelProvider and installing that again.
https://filamentphp.com/docs/3.x/panels/installation

I am unsure where to start debugging as there are no errors, and I can even see that it does indeed query the database for the correct model using clockwork.

**Reply by leandrocfe** [reply]
Could you share the project on github?

**Reply by sinnbeck** [reply]
Sadly not currently. I would need to try and recreate it in a new project. But one thing I noticed that might be a lead. There is no logout button. I am using fortify for logging in. Does that perhaps not work well with the panel? I have tried logging out but now I cannot seem to sign in anymore as filament wants to handle the login (it shouldnt)

Edit: Changed  `->login()` to return by own login page and that lets me login again. But perhaps there is some sort of thing my user is lacking? I login using ldap with the ldap-record package

**Reply by sinnbeck** [reply]
Did some more digging. Apparently the container actually is created but has a hidden class appended.
![image](https://github.com/filamentphp/filament/assets/13980140/355f80e2-2306-4adc-bb93-b48f70fe7241)

If I remove that, it actually shows up (more or less)! So I am thinking there must be some sort of error with the JS. I will keep investigating

![image](https://github.com/filamentphp/filament/assets/13980140/8a35a808-30b1-4b31-a8d6-ec5892340d86)

**Reply by sinnbeck** [reply]
Ok weird. I changed it to use another model.. It started working.. So I changed it back and it was still working.. I am clueless :/
